Formula/qrcp.rb

```diff
@@ -0,0 +1,16 @@
+class Qrcp < Formula
+  desc "Transfer files to and from your computer by scanning a QR code"
+  homepage "https://claudiodangelis.com/qrcp"
+  url "https://github.com/claudiodangelis/qrcp/archive/0.6.1.tar.gz"
+  sha256 "a42e6540b0293138b4cf5de2f0d8a988303b289c2eb012a67a80b3527c3c8c5f"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args
+  end
+
+  test do
+    system "#{bin}/qrcp", "--help"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.